include/glow/Support/CompilationLog.h

```diff
@@ -0,0 +1,62 @@
+/**
+ * Copyright (c) 2017-present, Facebook, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+#ifndef GLOW_SUPPORT_COMPILATIONLOG_H
+#define GLOW_SUPPORT_COMPILATIONLOG_H
+
+#include <string>
+#include "llvm/ADT/SmallString.h"
+#include "llvm/Support/CommandLine.h"
+
+
+
+namespace glow {
+
+extern  llvm::cl::opt<bool> dumpCompilationLogOpt;
+
+/// A class that defines the compilation scope. 
+///
+/// Example: 
+class CompilationScope {
+public:
+    /// A static stack that maintains the current scope stack.
+    static std::vector<std::string> scopeStack;
+private:
+    std::string scopeName;
```

Comment:
`scopeName` -> `scopeName_`.

Also, when is this used? It is initialized but never used after that.